src/orestandardtoken.js

```diff
@@ -19,12 +44,23 @@ async function getStandardTokenBalance(accountName, tokenSymbol, contractName) {
 
 // cleos push action cpu.ore transfer '["test1.apim", "test2.apim", "10.0000 CPU", "memo"]' -p test1.apim
 async function transferStandardToken(fromAccountName, toAccountName, tokenAmount, memo="", contractName) {
+  // Standard token transfer
   const { contract, options } = await this.contract(contractName, fromAccountName)
   await contract.transfer(fromAccountName, toAccountName, tokenAmount.toString(), memo, options);
 }
 
+// cleos push action cpu.ore transferfrom '["app.apim", "test1.apim", "test2.apim", "10.0000 CPU"]' -p app.apim
+async function transferfrom(approvedAccountName, fromAccountName, toAccountName, tokenAmount,contractName) {
+  // Standard token transfer
+  const { contract, options } = await this.contract(contractName, approvedAccountName)
+  await contract.transferfrom(approvedAccountName, fromAccountName, toAccountName, tokenAmount.toString(), options);
```

Comment:
Why do we need to pass in the approved account? The approved account is the one making the request.

Reply:
How is this different from the transfer method?

Reply:
Transfer method is the standard EOS transfer method and transferfrom is what we implemented  to have one account transfer on behalf of another.